src/clj/kixi/hecuba/data/measurements/calculations.clj

```diff
@@ -40,6 +40,42 @@
        (keep :value)
        (calculation-fn)))
 
+(defn morning?
+  "Returns true if measurement falls between 5:00 and 10:00.
+  Returns false otherwise."
+  [x]
+  (let [timestamp (tc/to-date-time (:timestamp x))
+        start     (time/update-timestamp! timestamp {:hour 5 :minutes 0 :seconds 0})
+        end       (time/update-timestamp! timestamp {:hour 10 :minutes 0 :seconds 0})]
+    (t/within? (t/interval start end) timestamp)))
```

Comment:
I'd like the splitting of the timestamp from the measurement separate from the checking of the period and move the checking of the period to k.h.time.

Something like this:

```clojure
k.h.measurements.calculations
(defn morning?
  "Returns true if measurement falls between 5:00 and 10:00.
  Returns false otherwise."
  [x]
  (-> x :timestamp tc/to-date-time time/morning?))
```

```clojure
k.h.time
(defn morning? [timestamp]
  (let [start     (time/update-timestamp! timestamp {:hour 5 :minutes 0 :seconds 0})
        end       (time/update-timestamp! timestamp {:hour 10 :minutes 0 :seconds 0})]
    (t/within? (t/interval start end) timestamp)))
```